seed/network.go

```diff
@@ -271,15 +272,15 @@ func (nv *NetworkView) reachabilityPruner() {
 		// addresses are reachable.
 		case newNode := <-nv.freshNodes:
 			go func() {
-				log.Infof("waiting to grab sema")
+				// log.Infof("waiting to grab sema")
 				<-searchSema
 
 				defer func() {
 					searchSema <- struct{}{}
-					log.Infof("sema returned")
+					// log.Infof("sema returned")
 				}()
 
-				log.Infof("got sema")
+				// log.Infof("got sema")
```

Comment:
Can be removed, was stray logging to debug something live in production...

Reply:
removed

Reply:
removed